README.md

```diff
@@ -1,5 +1,16 @@
 # Wortels
-This is the design repo for wortels, a specification for some random project.
+
+This is the design repo for Wortels - a novel design for a consensus computer.
+
+It preliminary takes learnings from Ethereum.
```

Comment:
Not sure what this sentence means.
Maybe: "It is intended to build on learnings from Ethereum, an existing blockchain computer."